### ISSUE TITLE
Using tab order to view the days of the schedule immediately changes the content

### ISSUE THREAD

**Issue by ADubhlaoich** [EuroPython/website#696]
## Description
**As an** attendee, **I want** to change the displayed date of the schedule, **so that** I can check what's happening

## Acceptance criteria
The page should not immediately load a new day when the dropdown for selecting a day receives focus through tab order, such as with a keyboard or adaptive switch.

Instead, a user should be able to tab through each day in the dropdown and escape it.

## Additional information
[WCAG 2.1 Keyboard Accessible](https://www.w3.org/WAI/WCAG21/Understanding/keyboard-accessible)
[WCAG 3.2.1 On Focus](https://www.w3.org/WAI/WCAG21/Understanding/on-focus)

**Comment by ADubhlaoich** [comment]
If you're using a device to click or press the dropdown for the day, you are presented with a list.

You can then select a day to change what the schedule is loaded, or exit the dropdown context.

This is not the case when navigating with a keyboard, and likely an adaptive switch. 

When the dropdown receives focus, selecting it does nothing, and navigating up or down immediately swaps to the next day.

This means someone would have to load and navigate the page 4 times just to get to information about Friday.

**Comment by patrick91** [comment]
@ADubhlaoich I was thinking of replacing the select with a standard list, what do you think of that? 😊

**Comment by ADubhlaoich** [comment]
> @ADubhlaoich I was thinking of replacing the select with a standard list, what do you think of that? 😊

That could work, but I think it's also a case of making the dropdown behave the same way when using a keyboard as when selected with another input device.

(Select is the generic nomenclature for click/press, since the latter two are dependent on platform).

**Comment by patrick91** [comment]
@ADubhlaoich on what browser/os do you have this issue with the select?

**Comment by ADubhlaoich** [comment]
I've reproduced this issue on both Firefox and Chrome.

Interestingly, on Chrome you can use the Return key to activate the dropdown as you would when using normal selection input, but it doesn't work on Firefox.

**Comment by patrick91** [comment]
@ADubhlaoich just tried on firefox and I get this behaviour:

- When using up/down arrows the select opens and you can choose any day (enter will trigger the value change and open the new day)
- When using left/right the value changes directly and you go to the next/previous day directly

I'm assuming this is working correctly, no?

**Comment by egeakman** [comment]
@ADubhlaoich is the issue still valid?

**Comment by artcz** [comment]
I'm going to close this for now – for 2025 the schedule looks and works different, and we no longer have the dropdown to change the days.
